tests/pcmjava/tools.vitruv.applications.pcmjava.tests.util/src/tools/vitruv/applications/pcmjava/tests/util/Java2PcmTransformationTest.java

```diff
@@ -865,10 +862,10 @@ protected OperationProvidedRole addImplementsCorrespondingToOperationProvidedRol
 		// ADDITION: Using Maven, changes run properly without the sleep, so it
 		// is removed by now.
 		// In Eclipse, it does not work without the sleep.
-		// try {
-		// Thread.sleep(5000);
-		// } catch (InterruptedException e) {
-		// }
+		try {
+			Thread.sleep(5000);
```

Comment:
This sleep operation should not be activated by default. It is only to be used if tests executed in Eclipse do not work, but not for CI.